### PR TITLE
docs: community docs

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds community docs to the `mdn` folder

### CODE REVIEW

File: files/en-us/mdn/community/contributing/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+We are an open community of developers, technical writers, and learners building resources for a better Web, regardless of brand, browser, or platform. Anyone can contribute, and each person who does contribute makes us stronger. Together we can continue to drive innovation on the Web to serve the greater good. It starts here, with you.
+
+[Join us!](https://github.com/mdn/mdn-community/)
+
+## What can I do to help?
+
+There are multiple avenues you can take to contribute to MDN, depending on your skill set and interests. Therefore, along with each task, we provide a short description and an approximate time each type of task typically takes.
+
+> If unsure what to do, you are always welcome to [ask for help](https://github.com/mdn/mdn-community/).
```
Comment (OnkarRuikar) [inline files/en-us/mdn/community/contributing/getting-started/index.md:19]:
```suggestion
> If unsure what to do, you are always welcome to [ask for help](https://chat.mozilla.org/#/room/#mdn:mozilla.org).
```
Chat room is a better place to ask simple queries.

File: files/en-us/mdn/community/contributing/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+> Also note that our small, but mighty docs team maintains this repo, to preserve our bandwidth, off topic conversations will be closed.
+
+## Primary contribution types
+
+We have created a [contributors task board](https://github.com/orgs/mdn/projects/25/views/1) to help you find contribution opportunities that will meaningfully impact the project. The board has an overview and separate views for specific contribution types.
```
Comment (OnkarRuikar) [inline files/en-us/mdn/community/contributing/getting-started/index.md:24]: `https://github.com/orgs/mdn/projects/25/views/1` is not visible to non members at the moment.

Only `https://github.com/orgs/mdn/projects/8/views/1` is.

Comment (schalkneethling) [reply files/en-us/mdn/community/contributing/getting-started/index.md:24]: Yup, that will be opened up soon.

File: files/en-us/mdn/community/contributing/getting-started/index.md (hunk 1 above)
Comment (schalkneethling) [reply files/en-us/mdn/community/contributing/getting-started/index.md:19]: Thank you for the suggestions, we are still thinking about how we can best manage this going forward.

**Review by Rumyra — APPROVED** [review]

**Review by dipikabh — APPROVED** [review]

File: files/en-us/mdn/community/contributing/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Getting started with MDN Web Docs
+slug: MDN/Community/Contributing/Getting_started
```
Comment (Rumyra) [inline files/en-us/mdn/community/contributing/getting-started/index.md:3]:
```suggestion
slug: MDN/Community/Contributing/Getting-started
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

**Comment by Rumyra** [comment]
Hmm seems like there's a lot of slug/folder inconsistencies... I'll check out this branch to fix

**Pull request merged**